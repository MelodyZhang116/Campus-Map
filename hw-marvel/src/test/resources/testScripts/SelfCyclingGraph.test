# test that the MarvelPaths does not store a self-cycling edge even if the graph contains self-cycling edge.

LoadGraph selfcycling SelfCyclingGraph.csv

ListChildren selfcycling n1