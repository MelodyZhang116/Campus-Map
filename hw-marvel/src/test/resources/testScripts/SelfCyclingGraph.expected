# test that the MarvelPaths does not store a self-cycling edge even if the graph contains self-cycling edge.

loaded graph selfcycling

the children of n1 in selfcycling are: n2(e1) n3(e1)